domains/dthesle.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "dthesle",
+    "email": "damienjarvs34@gmail.com"
+  },
+  "record": {
+    "CNAME": "dthesle.github.io"
```

Comment:
Little content displayed on site.

Reply:
What if I wanted to use this for subdomains that's connected to my main website?